query.sql

```diff
@@ -1,5 +1,5 @@
 \c nc_news_test
 
 Select * from articles
-where article_id = 1;
+where article_id = 4;
 
```

Comment:
This whole file should be .gitignored, as it's only a practice file it shouldn't reach the main branch, or even git hub at all.